frontend/src/components/task/task-report.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  FormControlLabel,
+  RadioGroup,
+  Radio,
+  Typography,
+  TextField
+} from '@material-ui/core'
+
+const Container = styled.div`
+  display: inline-block;
+  margin-right: 1rem;
+`
+
+class TaskReport extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = { open: false, reason: '', anotherReason: false }
+  }
+
+  static defaultProps = { visible: true }
+
+  static propTypes = {
+    visible: PropTypes.bool,
+    onClose: PropTypes.func,
+    reportTask: PropTypes.func
+  }
+
+  onChangeReason = (e) => {
+    const invalid = 'The issue is invalid, it was removed by the providers (Github, Bitbucket) and it\'s not available anymore'
```

Comment:
The reasons should be translated string as well, so you can create references like `report.reasons.reason1`